docs/store/README.md

```diff
@@ -36,19 +36,21 @@ combination of these reducers will make up your application state:
 // counter.ts
 import { Action } from '@ngrx/store';
 
-export const INCREMENT = 'INCREMENT';
-export const DECREMENT = 'DECREMENT';
-export const RESET = 'RESET';
+export const increment = 'INCREMENT';
```

Comment:
I think the all caps variable names are desired here.

Reply:
This is an archaic style that shouldn't infect ngrx.
https://angular.io/guide/styleguide#style-03-02

Reply:
Keep the constants in all caps